python/lsst/ip/isr/crosstalk.py

```diff
@@ -219,7 +296,10 @@ def subtractCrosstalk(exposure, badPixels=["BAD"], minPixelToMask=45000,
 
     ccd = exposure.getDetector()
     numAmps = len(ccd)
-    coeffs = ccd.getCrosstalk()
+    if crosstalkCoeffs is None:
```

Comment:
It's a legacy thing I don't want to break if I don't need to.  The previous version assumed coefficients could only be obtained by accessing the `ccd.getCrosstalk()` method.  This version allows it to be less restrictive, but since `subtractCrosstalk()` is independent, I didn't want that kind of call to cause problems.  I will admit that I can't find any issues with the new version from a scan of github.